README.md

```diff
@@ -1,5 +1,63 @@
 # Team 085 Group A Backend Application
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/f942cbe941324a0db0b4079cebb193a0)](https://app.codacy.com/gh/BuildForSDGCohort2/team-085-group-a-backend?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDGCohort2/team-085-group-a-backend&utm_campaign=Badge_Grade_Settings)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/f942cbe941324a0db0b4079cebb193a0)](https://app.codacy.com/gh/BuildForSDGCohort2/Team-085-Backend?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDGCohort2/team-085-group-a-backend&utm_campaign=Badge_Grade_Settings)
 
 Team 085 Group A Backend Application
+
+## Application Stack
+
+The Application Tech Stack includes:
+- A code editor you're comfortable with. I suggest [Visual Studio Code](https://code.visualstudio.com/) or [SublimeText](https://www.sublimetext.com/).
```

Comment:
There should be a total of 4 spaces after the unordered list indicator, that is:
-    Text here

Correct this for the entire document